<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>シリウス音声対話システム</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="chatDisplay">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>チャット履歴がここに表示されます...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLineEdit" name="messageInput">
        <property name="placeholderText">
         <string>メッセージを入力してください...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>送信</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelSpeechButton">
        <property name="text">
         <string>読み上げ停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="modelLayout">
      <item>
       <widget class="QLabel" name="modelLabel">
        <property name="text">
         <string>LLMモデル:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modelSelector">
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="manualAudioButton">
        <property name="text">
         <string>手動音声入力</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wakeWordButton">
        <property name="text">
         <string>ウェイクワード検知</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui></content>
<parameter name="filePath">/Users/kotaniryota/NLAB/oshaberi_sirius_control/src/main_window.ui